src/Raven.Server/Web/System/AdminDatabasesHandler.cs

```diff
@@ -523,6 +523,7 @@ public async Task GetRestorePoints()
         [RavenAction("/admin/restore/database", "POST", AuthorizationStatus.Operator)]
         public async Task RestoreDatabase()
         {
+            ServerStore.EnsureNotPassive();
```

Comment:
Move this inside the restore before saving the database record

Reply:
done